src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -268,10 +308,17 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
 
   render() {
     const { order, isCommittingMutation } = this.props
-    const { address, addressErrors, addressTouched } = this.state
+    const {
+      address,
+      addressErrors,
+      addressTouched,
+      phoneNumber,
+      phoneNumberError,
+      phoneNumberTouched,
+    } = this.state
     const artwork = get(
       this.props,
-      props => order.lineItems.edges[0].node.artwork
+      props => props.order.lineItems.edges[0].node.artwork
```

Comment:
Ohhh, was this broken before?

Reply:
Oh, hmm, I see. Yeah, you're right. That would work, but it isn't necessarily how it's intended to be used I think. This whole thing could also be something like

```
    const artwork = get(
      order,
      o => o.lineItems.edges[0].node.artwork
```

Really works either way though. I'll resolve this.